tests/rptest/tests/cluster_health_overview_test.py

```diff
@@ -41,7 +40,7 @@ def is_healthy():
 
         wait_until(is_healthy, 30, 2)
 
-    @cluster(num_nodes=5)
+    @cluster(num_nodes=5, log_allow_list=CHAOS_LOG_ALLOW_LIST)
```

Comment:
would it be enough to use `RESTART_LOG_ALLOW_LIST` redpanda is stopped gracefully with `stop_node`

Reply:
yes, should be enough, fixed!